### ISSUE TITLE
Error if there is not rightMenuViewController

### ISSUE THREAD

**Issue by ismaelgobernado** [dogo/AKSideMenu#9]
in the awakeFromNib i have:
```
self.contentViewController = self.storyboard!.instantiateViewController(withIdentifier: "ContentViewController")
        self.leftMenuViewController = self.storyboard!.instantiateViewController(withIdentifier: "LeftMenuViewController")
        self.rightMenuViewController = nil
```
this provokes an exception in  showLeftMenuViewController() line 288
```
 self.rightMenuViewController!.view.isHidden = true
```
accordingly to the declaration is an optional property so it can be nil 
```
var rightMenuViewController: UIViewController? 
```

thanks

**Comment by dogo** [comment]
Are you using the 1.3.0 version?

**Comment by ismaelgobernado** [comment]
I was using 1.2 , it works perfect with 1.3  thanks for your help

**Comment by Gerst20051** [comment]
made a comment on your initial commit about another occurrence when rightMenuViewController is nil. please go through and make sure all instances are covered so no crashes can occur.

**Comment by dogo** [comment]
@Gerst20051, please check the most recent code not the initial commit.

Thanks

**Comment by Gerst20051** [comment]
@dogo https://github.com/dogo/AKSideMenu/commit/68d9dc4b87adc18d9ac5bcb63aa62c710351624a

**Comment by Gerst20051** [comment]
@dogo it's present in the current code but that code hasn't changed since the initial commit.

**Comment by dogo** [comment]
@Gerst20051 , please comment in the current code showing what should be changed or open a PR

**Comment by Gerst20051** [comment]
@dogo i don't actually see the occurrence anymore. will have to check the code for the pod version i'm using. maybe the pod hasn't been updated yet.

**Comment by Gerst20051** [comment]
I think it might have been this line in version 1.2. https://github.com/dogo/AKSideMenu/blob/1.2/AKSideMenu/AKSideMenu.swift#L646

**Comment by dogo** [comment]
Yep, this is fixed since 1.3.0
